src/sync/outgoingSyncUtils.js

```diff
@@ -262,17 +262,22 @@ const generateSyncData = (settings, recordType, record) => {
       // Only sync out prescribers from this store.
       if (!record.fromThisStore) return null;
 
+      const initials = `${record.firstName[0] ?? ''}${record.lastName[0] ?? ''}`;
```

Comment:
`firstName` and `lastName` won't be null or undefined will they?

Reply:
There's a default value if unassigned, but I suppose you could technically explicitly set undefined! Cheers 80e316b